llvm/include/llvm/IR/IntrinsicsDirectX.td

```diff
@@ -30,6 +30,9 @@ def int_dx_handle_fromBinding
           [llvm_i32_ty, llvm_i32_ty, llvm_i32_ty, llvm_i32_ty, llvm_i1_ty],
           [IntrNoMem]>;
 
+def int_dx_typedBufferLoad
```

Comment:
Should I expect to find the `checkbit` version in here as well?

Reply:
No, I plan to add that in a follow up.